backend/shork/src/main/kotlin/Shork.kt

```diff
@@ -44,14 +45,20 @@ class Shork : IShork {
             val program = Program(player, shork)
             shork.addProgram(program)
 
-            val start = lastLocation
-            logger.info("Storing the program of player $player at location $start")
+            address =
+                if (internalSettings.randomSeparation) {
+                    (0 until internalSettings.coreSize).random()
+                    // @TODO: Test once functionality is inside InternalShork
+                } else {
+                    address + internalSettings.separation
+                }
+
+            val start = address
+            logger.info("Creating process for $player at address $address")
```

Comment:
I've found the first log message more descriptive of what's actually happening. This suggests a process is created, for example via a SPL